Formula/tbb.rb

```diff
@@ -17,16 +17,11 @@ class Tbb < Formula
 
   depends_on "cmake" => :build
   depends_on "swig" => :build
-  depends_on "python@3.9"
```

Comment:
Personally I think we're better off making this a `:build` dependency (as in `libxml2` and `z3`) than splitting this off into its own formula.